src/eei.h

```diff
@@ -69,7 +69,9 @@ class EthereumInterface {
     m_result.isRevert = false;
   }
 
+#if HERA_WAVM==OFF
```

Comment:
why is this conditionally compiled for WAVM only?

I think these methods should be protected anyway?

Reply:
Ok, I think it would then make sense to leave a note to remove it once the WAVM EEI implementation is more developed and actually calls those methods.

Reply:
Updated it with a comment.

My vote is to remove `protected:` entirely.